wGetStarted/L-Dev-SimpleDataSave.md

```diff
@@ -50,38 +64,50 @@ The System Parameter "Interject_RequestContext" will transfer contextual data to
 
 ## Setting up the Report
 
-Begin by opening up the report that was completed in the [Data Pull](/wGetStarted/L-Dev-CustomerAging.html#create-the-report). You will modify this report to set up the ReportSave.
+This example begins by setting up the Excel report. Open up the report that was completed in the [Data Pull](/wGetStarted/L-Dev-CustomerAging.html#create-the-report). You will modify this report to set up the ReportSave.
```

Comment:
`This example begins by setting up the Excel report.` may not really be needed.

Reply:
I left this as is. This page is a long one that is broken up in sections, the first being setting up the report.